lms/templates/fields/field_image.underscore

```diff
@@ -11,6 +11,7 @@
         <button class="u-field-remove-button" type="button">
             <span class="remove-button-icon" aria-hidden="true"><%= removeButtonIcon %></span>
             <span class="remove-button-title" aria-live="polite"><%= removeButtonTitle %></span>
+            <span class="sr"><%- gettext('profile image') %></span>
```

Comment:
This can't be hard-coded here. Pass a parameter `title` to the `ImageFieldView` and then use it here.